src/EnergyPlus/UtilityRoutines.cc

```diff
@@ -1206,26 +1206,33 @@ void ShowContinueErrorTimeStamp(EnergyPlusData &state, std::string const &Messag
     }
 
     if (len(Message) < 50) {
-        const auto m = Message + cEnvHeader + state.dataEnvrn->EnvironmentName + ", at Simulation time=" + state.dataEnvrn->CurMnDy + ' ' +
-                       CreateSysTimeIntervalString(state);
-        ShowErrorMessage(state, " **   ~~~   ** " + m, OutUnit1, OutUnit2);
+        const auto m = format("{}{}{}, at Simulation time={} {}",
+                              Message,
+                              cEnvHeader,
+                              state.dataEnvrn->EnvironmentName,
+                              state.dataEnvrn->CurMnDy,
+                              CreateSysTimeIntervalString(state));
+
+        ShowErrorMessage(state, format(" **   ~~~   ** {}", m), OutUnit1, OutUnit2);
         if (state.dataSQLiteProcedures->sqlite) {
             state.dataSQLiteProcedures->sqlite->updateSQLiteErrorRecord(m);
         }
         if (state.dataGlobal->errorCallback) {
             state.dataGlobal->errorCallback(Error::Continue, m);
         }
     } else {
-        const auto m = " **   ~~~   ** " + Message;
-        const auto postfix = " **   ~~~   ** " + cEnvHeader + state.dataEnvrn->EnvironmentName + ", at Simulation time=" + state.dataEnvrn->CurMnDy +
-                             ' ' + CreateSysTimeIntervalString(state);
-        ShowErrorMessage(state, m);
-        ShowErrorMessage(state, postfix, OutUnit1, OutUnit2);
+        const auto postfix = format("{}{}, at Simulation time={} {}",
+                                    cEnvHeader,
+                                    state.dataEnvrn->EnvironmentName,
+                                    state.dataEnvrn->CurMnDy,
+                                    CreateSysTimeIntervalString(state));
+        ShowErrorMessage(state, format(" **   ~~~   ** {}", Message));
+        ShowErrorMessage(state, format(" **   ~~~   ** {}", postfix), OutUnit1, OutUnit2);
         if (state.dataSQLiteProcedures->sqlite) {
-            state.dataSQLiteProcedures->sqlite->updateSQLiteErrorRecord(m);
+            state.dataSQLiteProcedures->sqlite->updateSQLiteErrorRecord(Message);
         }
         if (state.dataGlobal->errorCallback) {
-            state.dataGlobal->errorCallback(Error::Continue, m);
+            state.dataGlobal->errorCallback(Error::Continue, Message);
```

Comment:
I am changing this one. Currently it's writing to the SQL (and errorCallback) with the  `** ~~~ **` in it which isn't the case in the other ladder of the if block and is not wanted as far as I can tell